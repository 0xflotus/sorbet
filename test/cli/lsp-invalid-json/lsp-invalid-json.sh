echo "Content-Length: 1

" | main/sorbet --lsp 2>&1