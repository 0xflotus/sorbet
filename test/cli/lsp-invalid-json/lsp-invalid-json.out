Read: 

Last LSP request: `
` is not a valid json object
